db/offchaindb/db.go

```diff
@@ -0,0 +1,483 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package offchaindb
```

Comment:
Please document the new packages with `// package offchaindb ...` (or whatever package name it turns into).